lib/model/src/test/java/io/atlasmap/v2/JsonMarshallerTest.java

```diff
@@ -137,19 +138,40 @@ public void testFieldActions() throws Exception {
         }
 
         mapper.writeValue(new File("target/junit/" + testName.getMethodName() + "/" + "atlasmapping.json"),
-                atlasMapping);
+            atlasMapping);
         AtlasMapping generatedMapping = mapper.readValue(
-                new File("target/junit/" + testName.getMethodName() + "/" + "atlasmapping.json"), AtlasMapping.class);
+            new File("target/junit/" + testName.getMethodName() + "/" + "atlasmapping.json"), AtlasMapping.class);
 
         List<Action> generatedActions = ((Mapping) generatedMapping.getMappings().getMapping().get(0))
-                                            .getOutputField().get(0).getActions().getActions();
+            .getOutputField().get(0).getActions();
         assertEquals(actionsList.size(), generatedActions.size());
-        for (int i=0; i<actionsList.size(); i++) {
+        for (int i = 0; i < actionsList.size(); i++) {
             assertEquals(String.format(
-                    "Did you forget to add '%s' in ActionsJsonDeserializer/ActionsJsonSerializer?",
-                    actionsList.get(i).getClass().getName()),
-                    actionsList.get(i).getClass().getName(), generatedActions.get(i).getClass().getName());
+                "Did you forget to add '%s' in ActionsJsonDeserializer/ActionsJsonSerializer?",
+                actionsList.get(i).getClass().getName()),
+                actionsList.get(i).getClass().getName(), generatedActions.get(i).getClass().getName());
         }
     }
 
+    @Test
+    public void testNullActionDeserialization() throws Exception {
+
+        String content = "{\"io.atlasmap.v2.PropertyField\" : {\"jsonType\" : \"io.atlasmap.v2.PropertyField\",\"actions\" : [ {" +
+            "\"Split\" : { \"delimiter\": \"x\"}" +
+        "}]}}";
+        PropertyField f = mapper.readValue(content, PropertyField.class);
+//        System.out.println(mapper.writeValueAsString(f));
+
+        content = "{\"io.atlasmap.v2.PropertyField\" : {\"jsonType\" : \"io.atlasmap.v2.PropertyField\",\"actions\" : [ {" +
+            "\"Trim\" : null" +
+        "}]}}";
+        f = mapper.readValue(content, PropertyField.class);
```

Comment:
We still support deserializing actions using the old  `{ "<actionName>": { ... config .... } }` syntax.